docs/hub/spaces.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Documentation for Spaces
+---
+
+<h1>How to get started with Spaces</h1>
+
+> ⚠️ This feature is currently in private beta, reach out to [julien-c](https://huggingface.co/julien-c) if you'd like to try it out ⚠️
+
+## What are Spaces?
+
+Spaces are a simple way to host a ML demo app, directly on your user profile or your organization’s hf.co profile.
+
+We support two awesome SDKs that let you build cool apps in Python: **[Streamlit](https://streamlit.io/)** and **[Gradio](https://gradio.app/)**.
+
+**To get started**, simply click on [New Space](https://huggingface.co/new-space) in the top navigation menu, create a new repo of type `Space`, and pick your SDK:
+
+![/docs/assets/hub/new-space.gif](/docs/assets/hub/new-space.gif)
+
+Under the hood, we are storing your code inside a hf.co-hosted git repository, similar to what we're doing for models and datasets. So the same tools you're already used to (`git` and `git-lfs`) will also work for Spaces.
+
+We then deploy a containerized version of your code on our Infra, each time you commit. More details below!
+
+<!-- TODO(have someone record a Youtube demo of Spaces showcasing some cool apps already running, etc) -->
+
+## Should I use Streamlit or Gradio?
+
+We recommend you try both as they're both really awesome! 😎
+
+Streamlit's documentation is at https://docs.streamlit.io/, and Gradio's doc is https://gradio.app/docs.
+
+In the default environment, we're currently running version `"0.79.0"` of Streamlit and version `"2.0.9"` of Gradio.
+
+Our 2 cents:
+- **Gradio** is great if you want to build a super-easy-to-use interface to run a model from just the list of its inputs and its outputs. The Gradio team wrote a great [tutorial on our blog](https://huggingface.co/blog/gradio).
+- **Streamlit** gives you more freedom to build a full-featured Web app from Python, in a _reactive_ way (meaning that code gets re-run when the state of the app changes).
+
+You can also take a look at some sample apps on the [Spaces directory](https://huggingface.co/spaces) to make up your mind.
+
+<!-- Add screencap of listing directory -->
+
+Finally, we've been thinking of providing a way to run **custom apps**, for instance Python server code for the backend + a unified set of widgets/frontend JS code, or even custom Docker image serving. Do get in touch if you would like to build something more custom.
+
```

Comment:
```suggestion
## Can I create private Spaces?

Yes! Just like with models and datasets, a Hugging Face subscription will allow you to set a new Space as private when creating it. You can create a private Space for your account, or share a private Space within your organization. [Check our our plans](https://huggingface.co/pricing) to get started. 
```

Reply:
technically aren't restricted to Paid users, but we can still have this wording to promote Pro accounts (it's remaining in other parts of the doc too)